pallets/manta-pay/Cargo.toml

```diff
@@ -62,30 +58,33 @@ rand = { version = "0.8.4", default-features = false, optional = true }
 tempfile = { version = "3.3.0", optional = true }
```

Comment:
why do rand and tempfile have mentions in both deps and devdeps?

Reply:
`tempfile` and `rand` are optional in `dev`. So they are not available while `cargo test` or `cargo bench`.

Reply:
Ok I see, tests and the bin target use both packages